tools/sxml-srfi.scm

```diff
@@ -0,0 +1,440 @@
+                                   (string-index (car see-alsos) #\:))
+                      (car see-alsos)))
+             ((= (length see-alsos) 2)
+              (format "[[/srfi-~a/][SRFI ~a]] and [[/srfi-~a/][SRFI ~a]]"
+                      (string-take (car see-alsos)
+                                   (string-index (car see-alsos) #\:))
+                      (car see-alsos)
+                      (string-take (cadr see-alsos)
+                                   (string-index (cadr see-alsos) #\:))
+                      (cadr see-alsos)))
+             (else
+              (string-join
+               (append (map (lambda (see-also)
+                              (format "[[/srfi-~a/][SRFI ~a]]"
+                                      (string-take see-also
+                                                   (string-index see-also #\:))
+                                      see-also))
+                            (drop-right see-alsos 1))
+                       (list (format "and [[/srfi-~a/][SRFI ~a]]"
+                                     (string-take
+                                      (last see-alsos)
+                                      (string-index (last see-alsos) #\:))
+                                     (last see-alsos))))
+               ", "))))))
+
+(define (see-also->html-links see-alsos)
+  (if (null? see-alsos)
+      ""
+      (format
+       "<span class=\"see-also\">See also ~a.</span>"
+       (cond ((= (length see-alsos) 1)
+              (format "<a href=\"/srfi-~a/\">SRFI ~a</a>"
+                      (string-take (car see-alsos)
+                                   (string-index (car see-alsos) #\:))
+                      (car see-alsos)))
+             ((= (length see-alsos) 2)
+              (format "<a href=\"/srfi-~a/\">SRFI ~a</a> and <a href=\"/srfi-~a/\">SRFI ~a</a>"
+                      (string-take (car see-alsos)
+                                   (string-index (car see-alsos) #\:))
+                      (car see-alsos)
+                      (string-take (cadr see-alsos)
+                                   (string-index (cadr see-alsos) #\:))
+                      (cadr see-alsos)))
+             (else
+              (string-join
+               (append (map (lambda (see-also)
+                              (format "<a href=\"/srfi-~a/\">SRFI ~a</a>"
+                                      (string-take see-also
+                                                   (string-index see-also #\:))
+                                      see-also))
+                            (drop-right see-alsos 1))
+                       (list (format "and <a href=\"/srfi-~a/\">SRFI ~a</a>"
+                                     (string-take
+                                      (last see-alsos)
+                                      (string-index (last see-alsos) #\:))
+                                     (last see-alsos))))
+               ", "))))))
+
+(define (keyword->string keyword)
```

Comment:
The file `srfi-tools/data.scm` already defines `srfi-keyword-alist`.  Please find a way to refer to that rather than duplicating it.